core/lib/config/src/configs/use_evm_simulator.rs

```diff
@@ -0,0 +1,7 @@
+use serde::Deserialize;
+
+/// Configure whether to enable the EVM simulator on the stack.
+#[derive(Debug, Deserialize, Clone, PartialEq)]
+pub struct UseEvmSimulator {
```

Comment:
AFAIU, this config is never read properly in the main / external node initialization logic. Is it here for future use? I understand it's used as a surrogate for dynamically switching stuff in the codebase, but this is an anti-pattern and if it stays, it can be replaced with `std::env::var(_)` to not obscure the fact that it's a temporary hack.